### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by JinHai-CN**
### What problem does this PR solve?

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

### Type of change

- [x] Documentation Update

### CODE REVIEW

**Review by KevinHuSh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -58,10 +58,11 @@
 
 ## 📌 Latest Features
 
+- 2024-05-08 Integrate new LLM: DeepSeek.
```
Comment (writinwaters) [inline README.md]: Adds an LLM DeepSeek.

File: README.md [hunk 2]
```diff
@@ -58,10 +58,11 @@
 
 ## 📌 Latest Features
 
+- 2024-05-08 Integrate new LLM: DeepSeek.
 - 2024-04-26 Add file management.
```
Comment (writinwaters) [inline README.md]: Adds

File: README.md [hunk 3]
```diff
@@ -58,10 +58,11 @@
 
 ## 📌 Latest Features
 
+- 2024-05-08 Integrate new LLM: DeepSeek.
 - 2024-04-26 Add file management.
 - 2024-04-19 Support conversation API ([detail](./docs/conversation_api.md)).
```
Comment (writinwaters) [inline README.md]: Supports conversational API.

File: README.md [hunk 4]
```diff
@@ -58,10 +58,11 @@
 
 ## 📌 Latest Features
 
+- 2024-05-08 Integrate new LLM: DeepSeek.
 - 2024-04-26 Add file management.
 - 2024-04-19 Support conversation API ([detail](./docs/conversation_api.md)).
-- 2024-04-16 Add an embedding model 'bce-embedding-base_v1' from [BCEmbedding](https://github.com/netease-youdao/BCEmbedding).
-- 2024-04-16 Add [FastEmbed](https://github.com/qdrant/fastembed), which is designed specifically for light and speedy embedding.
+- 2024-04-16 Add an embedding model 'bce-embedding-base_v1' from [BCEmbedding](https://github.com/netease-youdao/BCEmbedding), and [FastEmbed](https://github.com/qdrant/fastembed) which is designed specifically for light and speedy embedding.
```
Comment (writinwaters) [inline README.md]: Adds xxx and FastEmbed, which xxx

File: README.md [hunk 5]
```diff
@@ -58,10 +58,11 @@
...
 ## 📌 Latest Features
 
+- 2024-05-08 Integrate new LLM: DeepSeek.
 - 2024-04-26 Add file management.
 - 2024-04-19 Support conversation API ([detail](./docs/conversation_api.md)).
-- 2024-04-16 Add an embedding model 'bce-embedding-base_v1' from [BCEmbedding](https://github.com/netease-youdao/BCEmbedding).
-- 2024-04-16 Add [FastEmbed](https://github.com/qdrant/fastembed), which is designed specifically for light and speedy embedding.
+- 2024-04-16 Add an embedding model 'bce-embedding-base_v1' from [BCEmbedding](https://github.com/netease-youdao/BCEmbedding), and [FastEmbed](https://github.com/qdrant/fastembed) which is designed specifically for light and speedy embedding.
+- 2024-04-16 
 - 2024-04-11 Support [Xinference](./docs/xinference.md) for local LLM deployment.
```
Comment (writinwaters) [inline README.md]: Same for the rest.

**Review by writinwaters — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -120,7 +119,7 @@
    $ chmod +x ./entrypoint.sh
    $ docker compose up -d
    ```
-   > Please note that running the above commands will automatically download the development version docker image of RAGFlow. If you want to download and run a specific version of docker image, please find the RAGFLOW_VERSION variable in the docker/.env file, change it to e.g. RAGFLOW_VERSION=v0.5.0, and then run the above command.
+   > Please note that running the above commands will automatically download the development version docker image of RAGFlow. If you want to download and run a specific version of docker image, please find the RAGFLOW_VERSION variable in the docker/.env file, change it to the corresponding version. For example, RAGFLOW_VERSION=v0.5.0, and then run the above commands.
```
Comment (writinwaters) [inline README.md]:
```suggestion
   > Please note that running the above commands will automatically download the development version docker image of RAGFlow. If you want to download and run a specific version of docker image, please find the RAGFLOW_VERSION variable in the docker/.env file, change it to the corresponding version, for example, RAGFLOW_VERSION=v0.5.0, and run the above commands.

```

**Review by writinwaters — APPROVED** [review]

**Pull request merged**